### PR TITLE
Update conda skeleton documentation to add link to Grayskull

### PR DESCRIPTION
**Pull request by travishathaway**
Adding a little admonition to let users know about Grayskull for building PyPI packages.

### Checklist - did you ...

- [ ] ~Add a file to the `news` directory ([using the template](../blob/main/news/TEMPLATE)) for the next release's release notes?~
     
- [ ] ~Add / update necessary tests?~
- [x] Add / update outdated documentation?

### CODE REVIEW

File: docs/source/user-guide/tutorials/build-pkgs-skeleton.rst [hunk 1]
```diff
@@ -7,6 +7,11 @@ Building conda packages with conda skeleton
    :local:
    :depth: 1
 
+.. warning::
+   For a better experience with building conda packages from PyPI pacakages, we recommend the usage
+   of `Grayskull <https://github.com/conda/grayskull>`. Please check out their project page on GitHub
```
Comment (travishathaway) [inline docs/source/user-guide/tutorials/build-pkgs-skeleton.rst:12]:
```suggestion
   of `Grayskull <https://github.com/conda/grayskull>`_. Please check out their project page on GitHub
```

File: docs/source/user-guide/tutorials/build-pkgs-skeleton.rst [hunk 2]
```diff
@@ -7,6 +7,11 @@ Building conda packages with conda skeleton
    :local:
    :depth: 1
 
+.. warning::
+   For a better experience with building conda packages from PyPI pacakages, we recommend the usage
```
Comment (dholth) [inline docs/source/user-guide/tutorials/build-pkgs-skeleton.rst:11]:
```suggestion
   For a better experience with building conda packages from PyPI packages, we recommend the usage
```

Comment (dholth) [reply docs/source/user-guide/tutorials/build-pkgs-skeleton.rst:11]: I would say "We recommend Grayskull, a newer alternative to conda-skeleton, to generate conda recipes for PyPI packages."

File: docs/source/user-guide/tutorials/build-pkgs-skeleton.rst [hunk 3]
```diff
@@ -7,6 +7,11 @@ Building conda packages with conda skeleton
...
+   of `Grayskull <https://github.com/conda/grayskull>`_. Please check out their project page on GitHub
```
Comment (travishathaway) [inline docs/source/user-guide/tutorials/build-pkgs-skeleton.rst:12]:
```suggestion
   We recommend `Grayskull <https://github.com/conda/grayskull>`_, a newer alternative to conda-skeleton,
   to generate conda recipes for PyPI packages. Please check out their project page on GitHub
```

File: docs/source/user-guide/tutorials/build-pkgs-skeleton.rst [hunk 4]
```diff
@@ -7,6 +7,11 @@ Building conda packages with conda skeleton
    :local:
    :depth: 1
 
+.. warning::
```
Comment (travishathaway) [inline docs/source/user-guide/tutorials/build-pkgs-skeleton.rst:10]:
```suggestion
.. tip::
```

File: docs/source/user-guide/tutorials/build-pkgs-skeleton.rst [hunk 5]
```diff
@@ -7,6 +7,11 @@ Building conda packages with conda skeleton
    :local:
    :depth: 1
 
+.. tip::
```
Comment (dholth) [inline docs/source/user-guide/tutorials/build-pkgs-skeleton.rst:10]: `.. admonition::` doesn't work does it

**Comment by travishathaway** [comment]
@dholth made some updates. Thanks for the suggestions.

**Review by kenodegard — APPROVED** [review]

**Pull request merged**